src/app/screens/SignUp/styles.module.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../../scss/variables/colors';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  justify-content: center;
+}
+
+.container-sign-up {
+  align-items: center;
+  background-color: $wild-sand;
+  border-top: 8px solid $cerulean;
+  display: flex;
+  flex-direction: column;
+  height: 634px;
+  justify-content: space-evenly;
+  padding: 0 25px;
+  width: 300px;
+
+  > * {
+    width: 100%;
+  }
+}
+
+.form {
+  > * {
+    width: 100%;
+  }
```

Comment:
same, you don't need this, these properties could are in the styles own of the component, and the button you could send a custom class. what do you think?

Reply:
Totally agree! that's the reason of the button can receive a custom class